docker/localstack/ready.d/init-aws.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Waiting for localstack to be ready...."
+sleep 5
```

Comment:
Is this really needed? I'd think localstack runs its ready.d scripts when it's up?

Reply:
Looks like it isn't needed. By the time this script is executed, localstack is ready: https://docs.localstack.cloud/references/init-hooks/

```
/etc
└── localstack
    └── init
        ├── boot.d           <-- executed in the container before localstack starts
        ├── ready.d          <-- executed when localstack becomes ready
        ├── shutdown.d       <-- executed when localstack shuts down
        └── start.d          <-- executed when localstack starts up
```